httpmw/ip_cognito_auth_test.go

```diff
@@ -104,7 +105,7 @@ func TestCognitoIP401(t *testing.T) {
 	called := false
 
 	router := gin.New()
-	router.Use(IpCognitoAuth(authTestIPRangesLarge, &cognitoIdentityProviderClientMock{}, authTestClientID))
+	router.Use(IpCognitoAuth(authTestIPRangesLarge, &cognitoIdentityProviderClientMock{}, authTestClientID, time.Minute*1))
```

Comment:
I'd be nice to add other cases when expiration = cache.NoExpiration and cache.DefaultExpiration

Reply:
I agree nice to have, will note that somewhere in to do list. Will probably move that to separate PR.